packages/website/pages/docs/resources/taiko-glossary.md

```diff
@@ -0,0 +1,95 @@
+## Anchor Transaction
+
+The first transaction in every Taiko L2 block to perform data validation and enable L1-to-L2 communication.
+
+## Bridge
+
+The Taiko bridge allowing users to bridge Ether and custom ERC-20 tokens from and to Taiko.
+
+## Commit Hash
+
+The Keccak-256 hash of a proposed block’s beneficiary and transaction list.
+
+## Cross-chain communication (messaging)
```

Comment:
i believe we can remove (messaging). we just refer to this as "Cross-chain communication" in the whitepaper